wrappers/services.go

```diff
@@ -778,6 +778,42 @@ func (es *ensureSnapServicesContext) ensureSnapJournaldUnits(quotaGroups *quota.
 	return nil
 }
 
+// ensureJournalQuotaServiceUnits takes care of writing service drop-in files for all journal namespaces.
+func (es *ensureSnapServicesContext) ensureJournalQuotaServiceUnits(quotaGroups *quota.QuotaGroupSet) error {
+	handleFileModification := func(grp *quota.Group, path string, content []byte) error {
+		old, modifiedFile, err := tryFileUpdate(path, content)
```

Comment:
nitpick: modifiedFile would be a good name for a variable containing the path or the name of the modified file.
```suggestion
		old, fileWasModified, err := tryFileUpdate(path, content)
```
Or even just "fileModified" could be OK.

Reply:
Done, thank you for the suggestion!